src/paths/components/create.yaml

```diff
@@ -0,0 +1,34 @@
+summary: Create a new component
+tags:
+  - components
+operationId: componentsCreate
+requestBody:
+  description: Body request that contains the ComponentDto object
+  required: true
+  content:
+    application/json:
+      schema:
+        $ref: "../../schemas/components/ComponentDto.yaml"
+responses:
+  '200':
+    description: Expected response to a valid request
+    content:
+      application/json:
+        schema:
+          $ref: "../../schemas/components/ComponentDto.yaml"
+  '403':
+    $ref: "../../responses/components/ResponseComponentForbidden.yaml"
+  '404':
+    description: Expected response to a non-existing component
+    content:
+      application/json:
+        schema:
+          $ref: "../../schemas/response/ErrorResponse.yaml"
+        example:
+          timestamp: "2021-02-09T07:57:01.062+0000"
+          status: 404
+          error: "Not Found"
+          message: "Component HttpExecuteRequest already exists"
```

Comment:
Not Found <-> Already Exists

Reply:
I copied pasted the response body when I make the request in postman. I have also seen this inconstency between the status code and the message. It seems that the REST API return 404 not found for an existing resource

Reply:
Okay, lets put this as a bug in the 'iesi' repository

Reply:
Okay, great